packages/js/onboarding/src/components/WooPaymentsMethodsLogos/index.tsx

```diff
@@ -218,24 +227,26 @@ export const WooPaymentsMethodsLogos: React.FC< {
 			{ maxShownElements < maxSupportedPaymentMethods && (
 				<div
 					className="woocommerce-woopayments-payment-methods-logos-count"
-					onClick={ () => setIsPopoverVisible( ! isPopoverVisible ) }
-					onMouseEnter={ () => setIsPopoverVisible( true ) }
-					onMouseLeave={ () => setIsPopoverVisible( false ) }
 					role="button"
 					tabIndex={ 0 }
+					onClick={ () => setPopoverVisible( ! isPopoverVisible ) }
+					onMouseEnter={ showPopover }
+					onMouseLeave={ hidePopoverDebounced }
 					onKeyDown={ ( event ) => {
 						if ( event.key === 'Enter' || event.key === ' ' ) {
-							setIsPopoverVisible( ! isPopoverVisible );
+							setPopoverVisible( ! isPopoverVisible );
 						}
 					} }
 				>
 					+ { maxSupportedPaymentMethods - maxShownElements }
 					{ isPopoverVisible && (
 						<Popover
-							position="top right"
+							placement="top-start"
 							className="woocommerce-woopayments-payment-methods-logos-popover"
+							focusOnMount={ false }
```

Comment:
We have nothing in the popover that is focusable so it best not to change the focused element.